### DISCUSSION TITLE
Ctrl + Shift + K opens command pallete

### DISCUSSION THREAD

**Discussion by aj185201** [community/community#15207] [Other Feature Feedback, Questions, & Ideas] [answered]
The documented shortcut for command pallete is `Ctrl + Alt + K`, but `Ctrl + Shift + K` opens command pallete, also `Ctrl + K `opens command pallete.

The Microsoft Edge browser (and maybe Google Chrome) already use the shortcuts so its kind of unwanted as in other tabs behaviour is different and on Github the behaviour is different.

- `Ctrl + K` is used to jump to address bar
- `Ctrl + Shift + K` is used to duplicate tab

Please fix this and don't use shortcuts which are already used by browsers.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @aj185201 👋🏼  Thanks for the feedback, we have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.

**Reply by aj185201** [reply]
ok great. but it would be great if the default shortcut was not a overridden shortcut @liligalante

**Reply by stuartleeks** [reply]
I agree with @aj185201. The help text says that `Ctrl+K` and `Ctrl+Alt+K` bring up the command palette. This suggests that `Ctrl+Shift+K` shouldn't, but it does.
It would be great if the behaviour matched the docs (i.e. `Ctrl+Shift+K` wasn't a shortcut for the command palette`
